coremltools/converters/mil/mil/ops/defs/_utils.py

```diff
@@ -295,28 +295,28 @@ def parse_einsum_equation(equation: str) -> Tuple[List]:
     output_str = input_output_str[1]
 
     inputs = input_str.split(',')
+    # TODO: remove here if arbitrary number operands for einsum is supported.
     assert len(inputs) == 2, "unsupported einsum equation {}".format(equation)
-    input1_str = inputs[0]
-    input2_str = inputs[1]
 
-    input1_vec = [-1 for i in range(len(input1_str))]
-    input2_vec = [-1 for i in range(len(input2_str))]
-    output_vec = [-1 for i in range(len(output_str))]
+    inouts = inputs + [output_str]
```

Comment:
rename it to `in_outs` might be more clear

Reply:
Thx! I renamed it.